### PR TITLE
Create Cirno's Perfect Bitmasks Classroom

### PR DESCRIPTION
**Pull request by parkjonggyeong18**

### CODE REVIEW

File: A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
+ #include <bits/stdc++.h>
+    using namespace std;
```
Comment (Waqar-107) [inline A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp:2]: align all the lines with `#include<bits/stdc++.h>`

File: A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    int main() {
+        ios_base::sync_with_stdio(false); cin.tie(NULL); cout.tie(NULL);
+     
+        int t;
+        cin >> t;
+        while (t--) {
+            long long x;
+            cin >> x;
+            long long ans = 0;
+            for (int i = 0; i < 33; i++) {
```
Comment (Waqar-107) [inline A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp:13]: add a new line before and after the for loop

File: A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                if ((x & (1LL << i))) {
+                    ans += (1LL << i);
+                    break;
+                }
+            }
+            if ((x ^ ans) > 0) {
```
Comment (Waqar-107) [inline A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp:19]: add a  newline before the `if`

File: A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                cout << ans << '\n';
+                continue;
+            }
+            for (int i = 0; i < 33; i++) {
+                if ((x & (1LL << i)) == 0) {
+                    ans += (1LL << i);
+                    break;
+                }
+            }
+            cout << ans << '\n';
```
Comment (Waqar-107) [inline A-set/1688A. Cirno's Perfect Bitmasks Classroom.cpp:29]: add a newline before the `cout`

**Review by Waqar-107 — CHANGES_REQUESTED** [review]
Please make the changes requested

**Comment by parkjonggyeong18** [comment]
I made a correction. Please merge.

2022년 10월 8일 (토) 오후 4:02, Waqar Hassan Khan ***@***.***>님이 작성:

**Review by Waqar-107 — APPROVED** [review]
Looks good 🚀

**Pull request merged**